SilasKenneth/fast-food-fast-ui
# [Finishes #160204351] User logged in homepage

#### What this pull request does
Creates a homepage for a logged in user
##### Description of the task to be completed
 Create a page where a logged in user can do stuff
    -  Add links for the logged in user
    - Add links for the user to choose other options
##### Relevant pivotal tracker story
[#160204351](https://www.pivotaltracker.com/n/projects/2194200/stories/160204351)
##### How to test
  - Clone the repo ``git clone https://github.com/SilasKenneth/fast-food-fast-ui.git``
  -  Go to the fast-food-fast-ui directory and head to UI folder, ```git checkout the ft-user-logged-in-homepage-160204351```
  - Open the additem.html file in your browser and that should work

#### Screenshot
![screencapture-localhost-3000-home-html-2018-08-31-20_51_17](https://user-images.githubusercontent.com/11555354/44934574-0d8eab00-ad76-11e8-8e5f-92bb9fdfb1a7.png)
